ApplicationLibCode/Application/Tools/Summary/RiaSummaryTools.cpp

```diff
@@ -40,6 +41,11 @@
 #include "RimSummaryPlot.h"
 #include "RimSummaryTable.h"
 #include "RimSummaryTableCollection.h"
+#include "RimWellPlotTools.h"
+#include "RimWellRftPlot.h"
+
+// The types used in this header file must be declared before including this header file
+#include "RiaObjectHandleTools.h"
```

Comment:
Maybe this should be part of caf, not Ria to avoid including the file at an "uncommon" place?

Reply:
Moved file to caf